pkg/executor/executor.go

```diff
@@ -408,7 +408,7 @@ func StartExecutor(ctx context.Context, logger *zap.Logger, functionNamespace st
 	if err != nil {
 		return err
 	}
-	go reaper.CleanupRoleBindings(ctx, logger, kubernetesClient, fissionClient, functionNamespace, envBuilderNamespace, time.Minute*30)
+	go reaper.CleanupRoleBindings(ctx, logger, kubernetesClient, fissionClient, defaultNs, time.Minute*30)
```

Comment:
Since this independent go routine, lets resolved all required variables from function itself. We shouldn't pass any arguments from top.

Reply:
done